lib/logqueue-fifo.c

```diff
@@ -539,8 +547,8 @@ log_queue_fifo_new(gint qoverflow_size, const gchar *persist_name)
     {
       INIT_IV_LIST_HEAD(&self->qoverflow_input[i].items);
       worker_batch_callback_init(&self->qoverflow_input[i].cb);
-      self->qoverflow_input[i].cb.user_data = self;
       self->qoverflow_input[i].cb.func = log_queue_fifo_move_input;
+      self->qoverflow_input[thread_id].cb.user_data = self;
     }
```

Comment:
It is corrected with a force push.
If there is no other review note, the PR should be merged